### DISCUSSION TITLE
Is there any way to get a user via their username?

### DISCUSSION THREAD

**Discussion by mmattbtw** [clerk/javascript#832] [Support request - DO NOT OPEN HERE] [answered]
Kinda like `clerk.users.getUser()` but instead of their ID, is there any way to get them by their username? I'm currently using `getUserList()` and filtering it through JavaScript which isn't ideal lol

**Comment by SokratisVidros** [comment, marked as answer]
@mmattbtw You can filter by username as described at the [Clerk Backend API reference docs](https://clerk.dev/docs/reference/backend-api/tag/Users#operation/GetUserList).

**Reply by mmattbtw** [reply]
🤯 completely missed this! thank you so much ❤️
